core/modules/update/update.compare.inc

```diff
@@ -417,14 +417,15 @@ function update_calculate_project_data($available) {
  * version (e.g., 5.x-1.5-beta1, 5.x-1.5-beta2, and 5.x-1.5). Development
  * snapshots for a given major version are always listed last.
  *
- * @param $project
- *   An array containing information about a specific project.
+ * @param $unused
+ *   Input is not being used, but remains in function for API compatibility
+ *   reasons.
  * @param $project_data
  *   An array containing information about a specific project.
  * @param $available
  *   Data about available project releases of a specific project.
  */
-function update_calculate_project_update_status($project, &$project_data, $available) {
+function update_calculate_project_update_status($unused, &$project_data, $available) {
```

Comment:
Since we're allowed to make changes to our API (though we're trying to limit them), let's just remove this first parameter entirely and update the calls to it.